src/server/log.js

```diff
@@ -14,7 +14,6 @@ function error(error) {
     }
     console.log(chalk.red.bold('SIM ERROR: ' + error));
     config.server && config.server.close();
-    process.exit(1);
```

Comment:
Ah, I missed that. But shouldn't the server still be responsible for closing itself in this scenario, rather than expecting the API consumer to do that?